### DISCUSSION TITLE
Reading wails.json

### DISCUSSION THREAD

**Discussion by wlwatkins** [wailsapp/wails#1934] [Q&A] [answered]
Hi, is there an method in runtime or something where I can access values from the wails.json or should I read it my self in the main.go?

**Comment by stffabi** [comment, marked as answer]
Currently there's no method for that, see also https://github.com/wailsapp/wails/discussions/1910#discussioncomment-3745469

**Reply by wlwatkins** [reply]
Thanks, I tried looking for already posted question but github came out empty. guess i did not use the right keywords. Thanks
